src/apps/code-editor/src/app/components/FileDrawer/components/LinkedContent/LinkedContent.js

```diff
@@ -39,14 +39,21 @@ export default function LinkedContent(props) {
                 </p>
 
                 <p>
-                  <Url
-                    className={styles.Link}
+                  <Link
+                    underline="none"
+                    color="secondary"
+                    sx={{
+                      alignItems: "baseline",
+                      display: "flex",
+                      margin: 0,
+                    }}
                     href={`${CONFIG.URL_PREVIEW_FULL}${item.web.path}`}
                     target="_blank"
                     title="Preview Item Webpage"
                   >
-                    <FontAwesomeIcon icon={faEye} /> <em>{item.web.path}</em>
-                  </Url>
+                    <FontAwesomeIcon icon={faEye} />{" "}
+                    <em sx={{ pr: 1 }}>{item.web.path}</em>
```

Comment:
Since em is not a mui component I don't think em does anything here. Additionally seems weird you need padding on the right side of the text